pkg/controller/direct/gkehub/featuremembership_controller.go

```diff
@@ -187,7 +187,12 @@ func (a *gkeHubAdapter) patchMembershipSpec(ctx context.Context) ([]byte, error)
 	// only change the feature configuration for the associated membership
 	mSpecs[a.membershipID] = *a.desired
 	feature.MembershipSpecs = mSpecs
-	op, err := a.hubClient.featureClient.Patch(a.featureID, feature).UpdateMask("membershipSpecs").Context(ctx).Do()
+	patchCall := a.hubClient.featureClient.Patch(a.featureID, feature)
```

Comment:
Q: was this documented anywhere?  It isn't the normal GCP pattern, so I was wondering how you figured this out...

Reply:
If you are asking about why doing patch to the membershipspec field, I added a comment in the code to explain.